metadatastore/commands.py

```diff
@@ -516,49 +924,15 @@ def _normalize_human_friendly_time(val):
     )
 
 
-def _normalize_object_id(kwargs, key):
-    """Ensure that an id is an ObjectId, not a string.
-
-    ..warning: Does in-place mutation of the search_dict
-    """
-    try:
-        kwargs[key] = ObjectId(kwargs[key])
-    except KeyError:
-        # This key wasn't used by the query; that's fine.
-        pass
-    except TypeError:
-        # This key was given a more complex query.
-        pass
-    # Database errors will still raise.
-
-
-def _get_mongo_document(document, document_cls):
-    """Helper function to get the mongo id of the mongo document of type
-    ``document_cls``
-
-    Parameters
-    ----------
-    document : mds.odm_templates.Document or str
-        if str: The externally supplied unique identifier of the mongo document
-    document_cls : object
-        One of the class objects from the metadatastore.odm_templates module
-        {RunStart, RunStop, Event, EventDescriptor, etc...}
-    """
-    if isinstance(document, Document):
-        document = document.uid
-    # .get() is slower than .first() which is slower than [0].
-    # __raw__=dict() is faster than kwargs
-    # see http://nbviewer.ipython.org/gist/ericdill/ca047302c2c1f1865415
-    mongo_document = document_cls.objects(__raw__={'uid': document})[0]
-    return mongo_document
-
-
 @_ensure_connection
-def find_run_starts(**kwargs):
+def find_runstarts(**kwargs):
     """Given search criteria, locate RunStart Documents.
 
     Parameters
     ----------
+    runstart : doc.Document or dict or str
```

Comment:
It is a little unclear why we are accepting doc.Document as a valid option here. If you already have the runstart document, why are you searching for it again? This should just be `str`